### PR TITLE
Update CHANGELOG

### PR DESCRIPTION
**Pull request by orelmisan**
Update the CHANGELOG with v0.3.0 release notes.

~~Depends on PR #244.~~

### CODE REVIEW

**Review by EdDev — CHANGES_REQUESTED** [review]
We should merge this CHANELOG from the 0.2 branch, to see a linear version history.

**Comment by orelmisan** [comment]
[Change](https://github.com/kiagnose/kiagnose/compare/a4917b49000a52e41f0fa046caca2f0046ca9874..c0bb8d8d8dc50a691504e496c3ccf883820246d9): Added version history from the 0.2 branch.

**Comment by orelmisan** [comment]
[Change](https://github.com/kiagnose/kiagnose/compare/c0bb8d8d8dc50a691504e496c3ccf883820246d9..5143c6b8f1f2aa6c22dcfccd9a2623d94486b629): Rebase

File: CHANGELOG [hunk 1]
```diff
@@ -1,5 +1,28 @@
 # Changelog
-## [0.3.0]
+## [0.3.0] - 2023-03-02
+### Breaking Changes
+ - VM latency checkup is now running as a standalone application and using Kiagnose packages as a library
```
Comment (EdDev) [inline CHANGELOG:4]: Can you just list here the diff from the last release? Some things here repeat from the ones below.

**Comment by orelmisan** [comment]
[Change](https://github.com/kiagnose/kiagnose/compare/5143c6b8f1f2aa6c22dcfccd9a2623d94486b629..404220d4eebdbcdbc95db3b00f5333f9c5aced8a): Removed from v0.3.0 items that were previously mentioned in v0.2.x.

File: CHANGELOG (hunk 1 above)
Comment (orelmisan) [reply CHANGELOG]: Done.

**Review by EdDev — APPROVED** [review]
Thanks!

**Pull request merged**